ocp_resources/resource.py

```diff
@@ -76,6 +83,29 @@ def _get_api_version(dyn_client: DynamicClient, api_group: str, kind: str) -> st
     return res.group_version
 
 
+@contextlib.contextmanager
+def get_retry_cluster_exceptions_client(
+    client: DynamicClient,
+    backoff_factor: float = 0.5,
+    retry_count: int = 5,
+    status_codes=None,
+) -> Any:
+    original_retries = client.configuration.retries
+    if not status_codes:
+        status_codes = set()
+    client.configuration.retries = urllib3.util.Retry(
```

Comment:
I think modifying the `configuration.retries` attribute after the client has been created has no effect.

AFAIK, you need to first create the configuration object, set its retry attribute and create the client from the configuration object.

Reply:
I do currently see the client is updated with the retry attributes. Are you saying that changes won't help? In that case wrapper would not be the right place for these changes.